pkg/arvo/app/spider.hoon

```diff
@@ -394,6 +463,25 @@
   :~  [%give %fact ~[/thread-result/[tid]] %thread-fail !>([term tang])]
       [%give %kick ~[/thread-result/[tid]] ~]
   ==
+++  thread-http-fail
+  |=  [=tid =term =tang]
+  ^-  (quip card ^state)
+  =-  (fall - `state)
+  %+  bind  
+    (~(get by serving.state) tid)
+  |=  [eyre-id=@ta output=mark]
+  :_  state(serving (~(del by serving.state) tid))
+  %+  give-simple-payload:app:server  eyre-id
+  ^-  simple-payload:http
+  :_  ~  :_  ~
+  ?.  ?=(http-error:spider term)
+    ((slog tang) 500)
```

Comment:
It would be nice to add a prefix tank to this saying it was an HTTP thread failure.  Not blocking.